### PR TITLE
Allow rsyslog to automatically manage state files

### PR DESCRIPTION
**Pull request by jbabe**
http://www.rsyslog.com/doc/v8-stable/configuration/modules/imfile.html#state-files

* Removes some of the unnecessary overhead of managing these names
ourselves

Here's an example of what rsyslog will do:
```bash
$ ls -1 /var/spool/rsyslog/
imfile-state:-var-log-nginx-access.log
imfile-state:-var-log-nginx-ssl-error.log
imfile-state:-var-log-rails-production.log
```

### CODE REVIEW

**Comment by adamrmelnyk** [comment]
🐘 🐘 🐘

File: defaults/main.yml [hunk 1]
```diff
@@ -14,3 +14,4 @@ loggly:
       - /var/log/nginx/access.log
     error_logs:
       - /var/log/nginx/error.log
+  action_queue_max_disk_space: 250m
```
Comment (MathieuGilbert) [inline defaults/main.yml]: Newline at end of file.

**Comment by MathieuGilbert** [comment]
💩

**Comment by jbabe** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**